sml_builder/templates/index.html

```diff
@@ -36,10 +36,16 @@
 {% endblock %}
 
 {% block pageContent %}
-<main id="main-content">
+<main id="main-content">        
+    {% if feature_1_active %}
+        {% set hero_title = "Feature 1 Activated" %}
```

Comment:
We may be better setting the titles via a config file, e.g "hero_title": "value", using a @app.context_processor method we can add these sort of values as variables to the jinja template directly so rather than these if else chains throughout we can just set hero_title = value

Reply:
I think that only works if the feature you are optioning lends itself to being declared as a set of JSON variables? I think we want clarity in the code where we are optioning functionality so it is clear that the page renders differently if feature_x is enabled.  My thinking is that this is really about functionality we want to integrate but not necessarily enable in prod. If we get to a point where we want to enable the functionality then it is removed as a feature from the file and just made part of the main code. Happy to discuss this through though.

Reply:
I agree, the main purpose of the feature optioning is to enable / disable sections of code rather than declaring what values variables should take. There will be repeated if-else blocks but they should be relatively temporary while we are working on a new feature.